### ISSUE TITLE
aria-activedescendant not reading out 1st option in JAWS

### ISSUE THREAD

**Issue by stevjo14** [angular-ui/ui-select#2035]
The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/angular-ui/ui-select/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description: 
aria-activedescendant doesn't read out/switch to the first option (index 0), and stays on index 1 (the second option) instead. 
e.g. aria-activedescendant="ui-select-choices-row-0-1"

### Link to minimally-working plunker that reproduces the issue: 
https://angular-ui.github.io/ui-select/demo-multiple-selection.html (all of the examples do the same thing)
**Screen Reader:** JAWS v17
**Browser:** Internet Explorer 11

### Version of Angular, UI-Select, and Bootstrap/Select2/Selectize CSS

**Angular:** 1.5.8

**UI-Select:** 0.19.8

**Bootstrap/Select2/Selectize CSS (if applicable):**

**Comment by Jefiozie** [comment]
Can you explain a bit more? I believe JAWS is software for reading a screen (at least that was my first google hit). How should it work and what should be adjusted to get this working?
